### PR TITLE
Cloud Logging fix

### LINKED ISSUE #351: Generated project (from dartfn) logs CloudEvent awkwardly
The generated file from `dartfn` [contains this code in `lib/functions.dart`](https://github.com/GoogleCloudPlatform/functions-framework-dart/blob/3a637639d44a09da554aa6eef70b3e3b112df0dc/dartfn/templates/cloudevent/lib/functions.dart#L12)

```dart
stderr.writeln(_encoder.convert(event));
```

Which renders awkwardly in the Cloud Run console with extraneous newlines and separate log statements.

Screen Shot 2022-10-31 at 12 59 24 PM

### PR DESCRIPTION
**Pull request by craiglabenz**
improves GCP logging conformance of generated code

New formatting:
Screen Shot 2022-11-04 at 11 43 14 AM

fixes #351

### CODE REVIEW

File: dartfn/templates/cloudevent/lib/functions.dart [hunk 1]
```diff
@@ -3,11 +3,18 @@ import 'dart:io';
 
 import 'package:functions_framework/functions_framework.dart';
 
-const _encoder = JsonEncoder.withIndent(' ');
+const _encoder = JsonEncoder();
```
Comment (craiglabenz) [inline dartfn/templates/cloudevent/lib/functions.dart:6]: This part fixes the log message being spread across multiple backend entries

File: dartfn/templates/cloudevent/lib/functions.dart [hunk 2]
```diff
@@ -3,11 +3,18 @@ import 'dart:io';
...
       .info('[CloudEvent] source: ${event.source}, subject: ${event.subject}');
-  stderr.writeln(_encoder.convert(event));
+  stderr.writeln(
+    _encoder.convert(
+      {
+        'message': event,
+        'serverity': LogSeverity.info,
+      },
+    ),
+  );
```
Comment (craiglabenz) [inline dartfn/templates/cloudevent/lib/functions.dart:19]: And this part improves conformance with https://cloud.google.com/logging/docs/agent/logging/configuration#special-fields

Comment (kevmoo) [reply dartfn/templates/cloudevent/lib/functions.dart:19]: sweet!

**Review by kevmoo — APPROVED** [review]

**Comment by kevmoo** [comment]
@craiglabenz – needed to run the generator. I did a force push to update the commit message, too. I'll land when green!

**Pull request merged**